data/parser/standard_object.go

```diff
@@ -148,12 +172,12 @@ func (s *StandardObject) ParseStringArray(key string) *[]string {
 	if !stringArrayValueOk {
 		arrayValue, arrayValueOk := rawValue.([]interface{})
 		if !arrayValueOk {
-			s.context.AppendError(key, ErrorTypeNotArray(rawValue))
+			s.AppendError(key, ErrorTypeNotArray(rawValue))
 			return nil
 		}
 
 		stringArrayValue = []string{}
-		parser, _ := NewStandardArray(s.context.NewChildContext(key), &arrayValue)
+		parser, _ := NewStandardArray(s.context.NewChildContext(key), &arrayValue, IgnoreNotParsed)
```

Comment:
Yes, I'm just utilizing the standard array parser here just to convert from an interface array to a string array. I don't want this parsing to indicate anything about whether it was actually parsed by the end code or not.